### ISSUE TITLE
Not aligned 

### ISSUE THREAD

**Issue by ibeey** [filebrowser/filebrowser#3385]
2.30 versions 
In list mode, the contents of the size and last modified columns are not aligned with the table header

<img width="960" alt="IMG_2024-08-03_10-07-24" src="https://github.com/user-attachments/assets/83d63908-0ce3-45d3-abe3-58bad74ce190">

**Comment by SilverSong3** [comment]
I know I could submit code to resolve the issue, but I believe I found a fairly quick solution.

Give just the p element with class="name" this line: margin-right: -3em;
Then just remove the "margin-right: 3em;" line from the p element with class="active name" and the text should line up no matter the screen size.

At least, that's the quick fix I found that seems to work with 2.30.0 that I used for finding this solution. I believe it stems from the image icon not being included in the flex parts that the text follows.

![Quick CSS Edit Fix](https://github.com/user-attachments/assets/bffcf567-4423-43c1-ad58-515ac169ac7d)
